website_sale_checkout_skip_payment/views/website_sale_template.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="confirmation" inherit_id="website_sale.confirmation">
+        <xpath expr="//div[@class='thanks_msg']//h2"
+               position="after">
+            <div class="alert alert-info" role="alert"
+                 t-if="website.checkout_skip_payment">
+                Our team will check your order and send you payment
+                information soon.
+            </div>
+        </xpath>
+        <xpath expr="(//div[@class='oe_cart']//h3)[2]"
+               position="attributes">
+            <attribute name="t-if">not website.checkout_skip_payment</attribute>
+        </xpath>
+        <xpath expr="(//div[@class='oe_cart']//table[@class='table'])[2]"
```

Comment:
It seems you are hiding the payment step. I'd appreciate a little comment line explaining this, to make the code more readable.